packages/core/src/lib/table/table-body.svelte

```diff
@@ -0,0 +1,30 @@
+<!--
+@component
+  
+A table body.
+```svelte
+<Table>
+  <TableBody></TableBody>
+</Table>
+```
+-->
+<svelte:options immutable />
+
+<script lang="ts">
+/**
+ * Custom table body style
+ * @example
+ * style="border-collapse"
```

Comment:
How does the intellisense look with this example? Should it be wrapped in a ``` block?

Reply:
with @example
<img width="185" alt="image" src="https://github.com/viamrobotics/prime/assets/31592018/e40a4f2a-5206-4adc-b5e8-2ef43d5a435b">

with ```
<img width="180" alt="image" src="https://github.com/viamrobotics/prime/assets/31592018/fe9bdaa5-cdd3-48fd-8ff3-0d4c3a2bd17a">

updating everything to ``` -- it looks cleaner